tests/integration/types/postgres.py

```
@@ -167,6 +168,17 @@ class FruitTypeWithPaginationAndOrderBy: ...
 class FruitFilter: ...
 
 
+def _fruit_sweeter_than(statement: Select[tuple[Fruit]], value: int, **_ctx: Any) -> Select[tuple[Fruit]]:
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_ctx`

(ANN401)